sbe-tool/src/main/java/uk/co/real_logic/sbe/generation/cpp/CppGenerator.java

```diff
@@ -2039,6 +2042,11 @@ private CharSequence generateMessageFlyweightCode(final String className, final
             "        return %8$s;\n" +
             "    }\n\n" +
 
+            "    SBE_NODISCARD static const std::string sbeSemanticVersion() SBE_NOEXCEPT\n" +
```

Comment:
This should be consistent with `sbeSemanticType` and use a `const char *`

Reply:
Done